.github/workflows/e2e_tests.yaml

```diff
@@ -114,6 +120,8 @@ jobs:
           kubectl create clusterrolebinding sdk-user-clusterqueue-creator --clusterrole=clusterqueue-creator --user=sdk-user
           kubectl create clusterrole localqueue-creator --verb=get,list,create,delete,patch --resource=localqueues
           kubectl create clusterrolebinding sdk-user-localqueue-creator --clusterrole=localqueue-creator --user=sdk-user
+          kubectl create clusterrole list-secrets --verb=get,list --resource=secrets
+          kubectl create clusterrolebinding sdk-user-list-secrets --clusterrole=list-secrets --user=sdk-user
```

Comment:
We would need to review the entire set of RBAC that are created there, but right now that's OK.